models/api_process_shipment.go

```diff
@@ -203,15 +203,15 @@ func (s *Shipment) CustomerReferences() []CustomerReference {
 	for idx, reference := range s.References {
 		switch idx {
 		case 0: // first index would always be the retailer name
-			customerReferences[idx] = CustomerReference{
+			customerReferences = append(customerReferences, CustomerReference{
 				CustomerReferenceType: CustomerReferenceTypeCustomerReference,
 				Value:                 sanitizeReferenceForFedexAPI(reference),
-			}
+			})
 		case 1: // second index would always be the order number
-			customerReferences[idx] = CustomerReference{
+			customerReferences = append(customerReferences, CustomerReference{
 				CustomerReferenceType: CustomerReferenceTypePurchaseOrder,
 				Value:                 sanitizeReferenceForFedexAPI(reference),
-			}
+			})
```

Comment:
This looks like it will properly handle two references - is the intent to support more than two, or just not break when there are more than two?

Reply:
It is to just support two.  I just want to fix the bug.

We tried this before with the fedex easypost integration.  A comma separated list of order numbers doesn't work because it only supports alphanumeric characters so we can't use a comma or space.

I don't think they support multiple references of the same type, so there's nothing we can do.